Framework/DataHandling/src/LoadSpiceXML2DDet.cpp

```diff
@@ -669,20 +669,17 @@ MatrixWorkspace_sptr LoadSpiceXML2DDet::createMatrixWorkspaceVersion2(
   }     // END-FOR (xml nodes)
 
   // Add the property to output workspace
-  for (std::map<std::string, std::string>::iterator miter = str_log_map.begin();
-       miter != str_log_map.end(); ++miter) {
+  for (auto &miter : str_log_map) {
```

Comment:
Could `miter` be replaced with something like `log_entry`? Similarly for line 676 and 680